src/sentry/feedback/usecases/create_feedback.py

```diff
@@ -240,6 +241,16 @@ def create_feedback_issue(event, project_id, source: FeedbackCreationSource):
     produce_occurrence_to_kafka(
         payload_type=PayloadType.OCCURRENCE, occurrence=occurrence, event_data=event_fixed
     )
+    if is_message_spam:
+        produce_occurrence_to_kafka(
```

Comment:
Not sure if this is the bug you're referring to but is it possible for this message to be processed before the `PayloadType.OCCURRENCE` message? would the group not exist in that case?

Reply:
yes, that's the bug i'm referring to, and that's what steve is fixing here: https://github.com/getsentry/sentry/pull/69466